packages/x-charts/src/ScatterChart/extremums.ts

```diff
@@ -24,18 +18,17 @@ export const getExtremumX: ExtremumGetter<'scatter'> = (params) => {
       const axisId = series[seriesId].xAxisId ?? series[seriesId].xAxisKey;
       return axisId === axis.id || (axisId === undefined && isDefaultAxis);
     })
-    .reduce(
-      (acc: ExtremumGetterResult, seriesId) => {
-        const seriesMinMax = series[seriesId].data.reduce(
+    .reduce<ExtremumGetterResult>(
+      (acc, seriesId) => {
+        const seriesMinMax = series[seriesId].data.reduce<ExtremumGetterResult>(
           (accSeries: ExtremumGetterResult, { x }) => {
```

Comment:
I added `<ExtremumGetterResult>` to avoid TS complaining about `number[]` not being assignable to `[number, number]`